### PR TITLE
489 link editing

### PR DESCRIPTION
**Pull request by DavieReid**

### CODE REVIEW

File: packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx [hunk 1]
```diff
@@ -78,7 +73,7 @@ export const InsertLinkDialog = () => {
         const selection = $getSelection();
         if ($isRangeSelection(selection)) {
           const textContent = selection.getTextContent();
-          setValues(values => ({ ...values, text: textContent }));
+          setValues(currentValues => ({ ...currentValues, text: textContent }));
```
Comment (tambo) [inline packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx:76]: currentValues isn't a thing here - should update`values` on line 55 where int with useState

**Review by tambo — CHANGES_REQUESTED** [review]
Needs comment about currentValues addressed or will break

File: packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx (hunk 1 above)
Comment (DavieReid) [reply packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx:76]: This is a functional `setState`.  `currentValues` is the previous state.

See --> https://react.dev/reference/react/useState#setstate

Comment (DavieReid) [reply packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx:76]: I will rename the variable to make it clearer

Comment (DavieReid) [reply packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx]: Updated

File: packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx [hunk 2]
```diff
@@ -118,35 +112,29 @@ export const InsertLinkDialog = () => {
...
+          </FormField>
+          <FormField validationStatus={errors?.text ? 'error' : undefined}>
+            <FormFieldLabel>Link Text</FormFieldLabel>
+            <Input value={values?.text} inputProps={{ name: 'text' }} onChange={handleChange} />
+            <FormFieldHelperText>{errors?.text}</FormFieldHelperText>
+          </FormField>
+        </div>
+      </DialogContent>
+      <DialogActions>
+        <ButtonBar>
```
Comment (ZarkoPernar) [inline packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx:131]: No need for ButtonBar in actions right now. Although the actions don't have the full functionality of the old button bar. But if this works well then maybe leave it. I haven't tried it myself.

**Review by ZarkoPernar — APPROVED** [review]

**Pull request merged**